src/init-ceph.in

```diff
@@ -447,6 +447,10 @@ for name in $what; do
 	    [ -n "$asok" ] && rm -f $asok
 	    [ -n "$post_stop" ] && do_cmd "$post_stop"
 	    [ -n "$lockfile" ] && [ "$?" -eq 0 ] && rm -f $lockfile
+	    # flush all data out of journal in background
```

Comment:
could you elaborate the rationale behind this change in the commit message? also, might want to fix the typo in the title of the commit message.

Reply:
I'll refind the commit message

Reply:
@tchaikov Previously, there's some unknown problem in ceph-v10.2.6 and this patch is used. I'm not sure what the problem is(long time ago) and whether the problem will be still hit in ceph master branch. Is it proper to still upstream this patch?

Reply:
@iotcg it depends on if we consider it's an improvement. if you are not sure if we have this problem and we cannot reproduce it anymore. i don't see any reason to fix it.

Reply:
Agree. Unclear change is not necessary.